extra/3-magic-8-ball.js

```diff
@@ -59,6 +91,17 @@ function shakeBall() {
   This function should expect to be called with any value which was returned by the shakeBall function.
 */
 function checkAnswer(answer) {
+  var shortMessage;
+  if (listOfAnswers.indexOf(answer) >= 0 && listOfAnswers.indexOf(answer) <= 4 ) {
+    shortMessage = "very positive";
+  } else if (listOfAnswers.indexOf(answer) >= 5 && listOfAnswers.indexOf(answer) <= 9) {
+    shortMessage = "positive";
+  } else if (listOfAnswers.indexOf(answer) >= 10 && listOfAnswers.indexOf(answer) <= 14) {
+    shortMessage = "negative";
+  } else if (listOfAnswers.indexOf(answer) >= 15 && listOfAnswers.indexOf(answer) <= 19) {
+    shortMessage = "very negative";
+  }
+  return shortMessage;
```

Comment:
Well done Roman, I like your code here, very clean, it was a slightly different approach than what I had done.